### DISCUSSION TITLE
Add beforeSend validation in Js

### DISCUSSION THREAD

**Discussion by fearovsky** [stayallive/wp-sentry#182] [Q&A] [answered]
Hello,
Im in little trouble with a javascript validation before send error to sentry. In php everything seems easy -have just to made a filter and everythings looks goood. Unfortunately i have no idea how to get the same effect in javascript.

Is there any inbuilt JS method to get instance of current sentry where can i set my rules ?

**Comment by stayallive** [comment, marked as answer]
I think the discussions in #175 are relevant here. It's a little harder and messier but certainly doable!

**Comment by fearovsky** [comment]
Yeah, the solution works perfectly! 😊 
The next question is about beforeAddRecordingEvent, because as I see in the Sentry documentation, it's in another class. 
So, I'll need to add some exceptions one more time.

Have you got inbuilt solution ?
